services/plugin/src/Plugin.tsx

```diff
@@ -27,23 +27,18 @@ const getPluginEntryPoint = ({
 export const Plugin = ({
     pluginSource,
     pluginShortName,
+    height,
+    width,
     ...propsToPassNonMemoized
 }: {
     pluginSource?: string
     pluginShortName?: string
+    height?: string
+    width?: string
```

Comment:
Allow `number` as a prop type?
```suggestion
    height?: number | string
    width?: number | string
```

Reply:
I don't know? I thought that it was more standard to define dimension as a string in the sense that it's supposed to include the unit: [https://developer.mozilla.org/en-US/docs/Web/CSS/dimension](https://developer.mozilla.org/en-US/docs/Web/CSS/dimension)

Are we seeing this as a compromise implementation? For example, update the prop type to take number, but not update the component documentation, such that we don't "advertise" this?

I guess that could be okay, as we probably don't want to promote numbers, since we're now relying on the iframe + browser implementation to convert `500` to `500px`.

Reply:
I don't actually see it as a compromise implementation; my intention is to match the behavior of the HTML `iframe` `height` attribute, which is a bit different than CSS requirements

It's a little hard to get formal documentation on what the accepted values actually are for iframe.height, but MDN at least indicates [it can be a number](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#height), and the autocomplete in VSCode indicates it can be a number or a string

Out of curiosity, when passing just `500` to the `height` attribute, was it formatted as a string? I wonder if that makes a difference...

Reply:
the mdn iframe reference says it’s a number (in pixels), but then the example shows a string-representation of a number, so I don't know at this point. It works for me to pass a number, a string-representation of a number, or a string with `px` or another unit specified. I don't really know what the limits are so I'll just update the type definition and leave the docs alone. 

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#height

Reply:
Why not update the docs? I think it makes sense to reflect the actual API

Does it still seem like a compromise to you? It seems standard to me since it mimics the `iframe.height` types, and based on our testing, seems well-supported -- I'm not sure why a number didn't work in one of your tests, but it seems a bit like a fluke